config-provisioning/src/main/java/com/yahoo/config/provision/zone/ZoneApi.java

```diff
@@ -18,5 +18,6 @@
 
     CloudName getCloudName();
 
+    /** Returns the region name within the cloud, e.g. 'us-east-1' */
```

Comment:
This comment does not help - more context is needed, e.g. "'us-east-1' for AWS"

Reply:
It applies for all current clouds...

Reply:
Fixed.